### PR TITLE
My apps

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Add ‘my apps’ link to main nav
Remove search specific breakpoint and use medium instead

## QA

- Run ./run to get started and go to http://0.0.0.0:8015/
- Check that the my apps link is on the right hand side but not visible on mobile
- Check that the search snaps at breakpoint-medium

### CODE REVIEW

**Comment by grahambancroft** [comment]
[Add ‘my apps’ link to main nav](https://trello.com/c/KBaXSt5p/3235-add-my-apps-link-to-main-nav)

File: templates/includes/base/header.html [hunk 1]
```diff
@@ -11,19 +11,23 @@
         <img src="{{ ASSET_SERVER_URL }}2196e362-search-white.svg" class="header-search__image" alt="Perform a search" />
     </button>
 
-    <nav id="navigation" class="p-navigation__nav">
+    <nav id="navigation" class="p-navigation__nav u-float--right">
         <span class="u-off-screen">
             <a href="#main-content">Jump to main content</a>
         </span>
+        <ul class="p-navigation__links">
+            <li class="p-navigation__link"><a class="" href="https://myapps.developer.ubuntu.com/">My Apps</a></li>
```
Comment (anthonydillon) [inline templates/includes/base/header.html]: Remove empty class on a tag.

Comment (anthonydillon) [reply templates/includes/base/header.html]: Should this be external?

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Some questions in the code. QA ok but how do you get to My Apps on small screens?

File: templates/includes/base/header.html (hunk 1 above)
Comment (grahambancroft) [reply templates/includes/base/header.html]: It should really, I’ve added the class and tweaked the SASS

**Review by anthonydillon — APPROVED** [review]
Code :+1:

**Pull request merged**